GTG/gtk/browser/taskbrowser.ui

```diff
@@ -8,529 +8,176 @@
+                    <property name="icon-name">edit-find-symbolic</property>
+                    <property name="icon-size">1</property>
                   </object>
-                  <packing>
-                    <property name="expand">False</property>
-                    <property name="homogeneous">True</property>
-                  </packing>
                 </child>
               </object>
               <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="position">0</property>
+                <property name="pack_type">end</property>
+              </packing>
+              <style>
+                <class name="image-button"/>
+              </style>    
+            </child>
+            <child>
+              <object class="GtkBox" id="extras_box">
+              <property name="visible">True</property>
+              <property name="can_focus">False</property>
+              <property name="has_focus">False</property>
+              <property name="is_focus">False</property>
+              <property name="spacing">6</property>
+              <child>
+                <object class="GtkButton" id="plugins">
+                  <property name="visible">True</property>
+                  <property name="label">Plugins</property>
+                  <property name="sensitive">True</property>
+                  <property name="tooltip_text" translatable="yes">Add plugins</property>
+                  <signal name="clicked" handler="on_edit_plugins_activate" swapped="no"/>
+                  <property name="valign">center</property>
+                </object>
+                <style>
+                  <class name="text-button"/>
+                </style>
+              </child>
+              <child>
+                <object class="GtkButton" id="synchronisation">
+                  <property name="visible">True</property>
+                  <property name="label">Synchronisation</property>
+                  <property name="sensitive">True</property>
+                  <property name="tooltip_text" translatable="yes">Synchronisation Services
+                     </property>
+                  <signal name="clicked" handler="on_edit_backends_activate" swapped="no"/>
+                  <property name="valign">center</property>
+                </object>
+                <style>
+                  <class name="text-button"/>
+                </style>
+              </child>
+              </object>
+              <packing>
+                  <property name="pack_type">end</property>
               </packing>
             </child>
           </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">False</property>
-            <property name="position">1</property>
-          </packing>
         </child>
+    <child>
```

Comment:
please, indent this file properly. I'm sure your editor might have support for doing it automatically